cmd/bnsd/app/app.go

```diff
@@ -59,6 +59,7 @@ func Chain(authFn x.Authenticator, minFee coin.Coin) app.Decorators {
 		msgfee.NewAntispamFeeDecorator(minFee),
 		msgfee.NewFeeDecorator(),
 		batch.NewDecorator(),
+		utils.NewActionTagger(),
```

Comment:
I think `utils` is [a bad package name](https://blog.golang.org/package-names#TOC_5). How about changing it to something else or moving action tagger to a different package?

Reply:
Good point. I would be happy for an issue/PR to rename utils, *for all items*.
Right now, it is a set of little helpers that don't deserve a full package.

Can you propose a new name in an issue, then if people like it, just make a PR to move it